src/index.html

```diff
@@ -1,12 +1,301 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Flex:opsz,wght@8..144,500&family=Roboto:wght@500&display=swap" rel="stylesheet"> -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Flex:opsz,wght@8..144,500&family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#"><img src="images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-li">
+            <a href="#apple" class="is-active nav__list-li-link">Apple</a>
+          </li>
+          <li class="nav__list-li">
+            <a href="#samsung" class="nav__list-li-link">Samsung</a>
+          </li>
+          <li class="nav__list-li">
+            <a href="#smartphones" class="nav__list-li-link">Smartphones</a>
+          </li>
+          <li data-qa="nav-hover" class="nav__list-li">
+            <a href="#laptops&computers" class="nav__list-li-link"
+             data-qa="nav-hover">Laptops & Computers</a>
+          </li>
+          <li class="nav__list-li">
+            <a href="#gadgets" class="nav__list-li-link">Gadgets</a>
+          </li>
+          <li class="nav__list-li">
+            <a href="#tablets" class="nav__list-li-link">Tablets</a>
+          </li>
+          <li class="nav__list-li">
+            <a href="#photo" class="nav__list-li-link">Photo</a>
+          </li>
+          <li class="nav__list-li">
+            <a href="#video" class="nav__list-li-link last_link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card"
+           data-qa="card">
+        <div>
+          <img class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock">
```

Comment:
```suggestion
          <img 
            class="card__image"
            src="images/imac.jpeg"
            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
          >
```
Please fix all cases